content/20.treatments.md

```diff
@@ -93,16 +93,29 @@ A list of current studies and their results, using carefully the information req
+
+### Current Evidence
+Remdesivir was used on four Covid-19 patients under compassionate use guidelines. All were in late stages of Covid-19 infection and the results have been mixed.  More recently Remdesivir entered controlled clinical trials.  As of March 2020, there are six clinical trials underway to treat Covid-19 patients at both early and late stages of infection and in combinations with other drugs (https://www.nature.com/articles/s41422-020-0282-0). (NCT04292730, NCT04292899, NCT04280705, 2020-000936-23, NCT04252664, NCT04257656). 
 Previous works already showed that Remdesivir was effective in the treatment of Covid-19 similar viral pathogens, such as SARS-Cov and MERS-Cov, in animal models, while a recent study showed that Remdesivir resulted in 100% protection against the Ebola virus in human liver Cancer cell line, which is also sensitive to the novel Corona virus [@doi:10.1038/s41422-020-0282-0]. 
 Upon FDA approval Gilead enrolled about a 1000 patients from COVID-19 hit countries for two phase III clinical trials to test the efficacy of the drug in adults. 
 Specifically, one trial will test 400 patients with severe COVID-19 symptoms, while the second trial will randomise the remaining amount of patients with mild symptoms. 
 In both studies, patients will receive the same Remdesivir dose for 10 days (Gilead.com). 
 Remdesivir is also currently involved in a trial to test the drug against Ebola virus since 2015. 
 However, in August 2019, after a large phase III trial in the Democratic Republic of Congo, the drug was declared to be less effective compared to other monoclonal antibody in use (Biopharmadive.com). 
 
+### Summary
+Remdesivir is a major drug candidate since it attacks the virus with high potency and known mechanism.  Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.  Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.  Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas Covid-19 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against Covid-19.    In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens. They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.   
```

Comment:
```suggestion
Remdesivir is a major drug candidate since it attacks the virus with high potency and known mechanism.
Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.
Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.
Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas Covid-19 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against Covid-19.
In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens.
They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.   
```